objects/obj_prompt/Alarm_0.gml

```diff
@@ -1,4 +1,9 @@
-/// @description Destroy me and do things
+// Alarm[0] event - after fade-out is complete, reset the object
 
-// destroy me
-//instance_destroy();
\ No newline at end of file
+// If the fade-out is done, you can either deactivate the object or reset its properties
+// Example: Resetting the object to prepare it for re-entry (make it visible again)
+fadeMe = FADE_IN;  // Set back to fade-in state so it can reappear
+image_alpha = 0;    // Reset alpha to make it invisible
+shift = 1;          // Reset bobbing direction to default
+alarm[1] = 10;      // Reset bobbing alarm to continue the bobbing effect
```

Comment:
shouldn't this be `alarm[0]`

Reply:
but bobbing alarm is in alarm[1], so should i be moving the bobbing effect to alarm[0] then?

Reply:
lol code acts weird if taken out so it must remain